_pages/dev/connecting-to-other-systems.md

```diff
@@ -1,74 +1,70 @@
-- **Facade layer**: The facade layer hides the complexity of the in-memory data store (ngrx). This layer is designed to simplify your development, and let you focus on custom view logic.
-- **In-memory Store (ngrx)**: Spartacus uses ngrx for state management. Ngrx is considered complex and therefor you can use the facade layer.
-- **Back end connector**: The back end connector is called by ngrx _effects_, and return the response from the back end in the required UI model. The connector delegates to an adapter, which interacts with a back end system.
+- **UI components:** The UI layer is only concerned with the view logic of the UI. The UI components observe data provided by the facade layer.
+- **Facade layer:** The facade layer hides the complexity of the in-memory data store (NgRx). This layer is designed to simplify your development, and let you focus on custom view logic.
+- **In-memory store:** Spartacus uses NgRx Store for state management. NgRx is considered complex, and it is recommended that you use the facade layer.
+- **Back-end connector:** The back-end connector is called by NgRx effects, and returns the response from the back end in the required UI model. The connector delegates to an adapter, which interacts with a back-end system.
 
-![component data binding]({{ site.baseurl }}/assets/images/component-data-binding.png)
+![component data binding]({{ site.baseurl }}/assets/images/data-binding-architecture.png)
 
-While this is a pretty complex setup, you don't need to worry about most layers. When you want to connect a UI component to an alternative data source, you would customize some low level layers without being concerned with the facade layer or data store. Only if alternative client-side business logic is required, you would provide additional logic, most likely close to the UI layer.
+Although this is a fairly complex setup, you do not need to worry about most layers. When you want to connect a UI component to an alternative data source, you can customize some low-level layers without being concerned with the facade layer or the data store. Only if alternative client-side business logic is required would you then provide additional logic (most likely, close to the UI layer).
 
 ## Connector Logic
 
-The Connector logic sits between the in-memory data store (ngrx) and the back end. A specific connector per domain is used to offload the connection to a back end system. For example, the product connector takes care of loading product details.
+The connector logic sits between the in-memory data store and the back end. A specific connector is used for each domain to offload the connection to a back-end system. For example, the product connector takes care of loading the product details.
 
-To provide optimal flexibility, there are three entities involved to connect a back end system:
+To provide optimal flexibility, there are three entities involved in connecting to a back-end system: a connector, an adapter, and a converter.
```

Comment:
Can we add:
Not all entities are necessarily involved when you adapt a third party system.

Reply:
Done.